static/js/zamboni/themes_review_templates.js

```diff
@@ -43,16 +43,18 @@ return __p;
 
 /*
 _.template(`
-<script type="text/template" id="queue-search-row-template">
   <tr class="addon-row">
     <td class="app-name">
       <span class="addon-id"><%= id %></span>
       <a href="<%= review_url %>"><%- name %></a>
     </td>
-    <td><a href="<%= reviewer %>"><%= reviewer %></a></td>
+    <td>
+      <% if (typeof reviewer !== "undefined") { %>
+        <a href="<%= reviewer %>"><%= reviewer %></a>
```

Comment:
Hmm, yeah I wondered that too - I'm not sure exactly what reviewer is. Especially since it's used for the url too.